tools/cli/dev-bundle-bin-helpers.js

```diff
@@ -41,3 +56,31 @@ exports.getEnv = function () {
 
   return env;
 };
+
+exports.spawn = function (command, args) {
+  var child = require("child_process").spawn(command, args, {
+    stdio: "inherit",
+    env: exports.getEnv()
+  });
+
+  require("./flush-buffers-on-exit-in-windows.js");
+
+  return child;
+};
+
+exports.npmInstall = function () {
+  var nodeModulesPath = path.join(__dirname, "..", "..", "node_modules");
+  try {
+    var stat = fs.statSync(nodeModulesPath);
+    if (stat.isDirectory()) {
```

Comment:
If `meteor-tools` is installed with npm v3, it's entirely possible that it won't have a `node_modules` directory because all of its dependencies were hoisted to the top level, so this check will have to be improved.